frontend/lib/src/components/elements/ArrowVegaLiteChart/styled-components.ts

```diff
@@ -220,3 +220,13 @@ export const StyledVegaLiteChartContainer =
       },
     })
   )
+
+export const StyledChartWrapper = styled.div<{
```

Comment:
**suggestion:** Seems like we have this component defined 3 times (ArrowVega, GraphViz, ImageList), with only 1 difference between them (ImageList has a `maxWidth` set). Can we put this into a shared spot and reuse them?

Reply:
I was thinking the same thing, and said that if someone brings it up during PR review, I'd do it. So time to do it!

Reply:
and i don't think the max-width would hurt the other ones either... 🤔